BraveWallet/Crypto/Onboarding/RecoveryPhraseGrid.swift

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import SwiftUI
+import Algorithms
+
+/// Displays a grid of recovery words
+struct RecoveryPhraseGrid<Word: Hashable, ID: Hashable, Content: View>: View {
+  @Environment(\.sizeCategory) private var sizeCategory
+  
+  var data: [Word]
+  var id: KeyPath<Word, ID>
+  var spacing: CGFloat
+  var content: (Word) -> Content
+  
+  init(
+    data: [Word],
+    id: KeyPath<Word, ID>,
+    spacing: CGFloat = 8.0,
+    @ViewBuilder content: @escaping (Word) -> Content
+  ) {
+    self.data = data
+    self.id = id
+    self.spacing = spacing
+    self.content = content
+  }
+  
+  private var numberOfColumns: Int {
+    sizeCategory.isAccessibilityCategory ? 2 : 3
+  }
+  
+  /// The data chunked into columns for iOS 13 APIs
+  private var gridData: [[Word]] {
+    data.chunks(ofCount: numberOfColumns).map(Array.init)
+  }
+  
+  var body: some View {
+    let columns: [GridItem] = (0..<numberOfColumns).map { _ in .init(.flexible()) }
+    LazyVGrid(columns: columns, spacing: spacing) {
+      ForEach(data, id: id) { entry in
+        content(entry)
+      }
+    }
+  }
+}
+
+struct RecoveryPhraseGrid_Previews: PreviewProvider {
+  static let data: [String] = [
```

Comment:
is this a real recovery phrase? is there any chance a user would somehow set up a wallet with this phrase if so?

we want to avoid the same thing that happened with sync where users accidentally set up sync with example codes

Reply:
@diracdeltas This is not a valid phrase, double checked with desktop.